src/main/java/net/skinsrestorer/shared/storage/SkinStorage.java

```diff
@@ -320,11 +324,16 @@ public Object getSkinData(String name, boolean updateOutdated) {
     }
 
     public Object getSkinData(String name) {
-        return this.getSkinData(name, true);
+        return this.getSkinData(name, false);
```

Comment:
I think this is a unwanted change, we would always want to run the update check on default and have it only not update in some occasions like display (example: skinsgui)

Reply:
Agreed. Will revert.